### ISSUE TITLE
Doubts in Processing High-throughput Sequencing Data and Microarray Data

### ISSUE THREAD

**Issue by fangshiyu1910426** [l-magnificence/Mime#115]
Hello,may I ask if it is possible to use high-throughput sequencing(RNA-seq)data as the training set and microarray data as the validation set for machine learning using Mime?Will the different types of data affect the results?If it is possible to do so,how should I standardize my expression matrix? Thanks.

**Comment by l-magnificence** [comment]
RNA-seq and microarray data are all fine for Mime. However, we suggest that each dataset should have high quality with similar sequencing depth and gene quantity and be processed by same normalization method. which may impact the accuracy and reliability of the model.

**Comment by fangshiyu1910426** [comment]
Thank you very much for your answer.I would like to know further that microarray data often uses the normalization method of log2(x+1),while RNA-seq often uses the normalization method of log2(tpm+1).May I ask if I can use log2(tpm+1)RNA-seq as the training set and log2(x+1)microarray data as the validation set?Will the mime1 package further perform z-score normalization on the training and validation datasets input as log2(x+1),such as data<-scale(data,center=TRUE,scale=TRUE)?Thank you.

**Comment by fangshiyu1910426** [comment]
> RNA-seq and microarray data are all fine for Mime. However, we suggest that each dataset should have high quality with similar sequencing depth and gene quantity and be processed by same normalization method. which may impact the accuracy and reliability of the model.

Thank you very much for your answer.I would like to know further that microarray data often uses the normalization method of log2(x+1),while RNA-seq often uses the normalization method of log2(tpm+1).May I ask if I can use log2(tpm+1)RNA-seq as the training set and log2(x+1)microarray data as the validation set?Will the mime1 package further perform z-score normalization on the training and validation datasets input as log2(x+1),such as data<-scale(data,center=TRUE,scale=TRUE)?Thank you.

**Comment by l-magnificence** [comment]
You need to perform normalization yourself before running Mime. Normalization method of log is ok.

**Comment by fangshiyu1910426** [comment]
> You need to perform normalization yourself before running Mime. Normalization method of log is ok.

Thank you very much for your help.
